bitmind/validator/cache/base_cache.py

```diff
@@ -144,15 +148,15 @@ async def _run_compressed_updater(self) -> None:
                 bt.logging.error(f"Error in compressed cache update: {e}")
                 await asyncio.sleep(60)
 
-    def _refresh_compressed_cache(self, n_per_source) -> None:
+    def _refresh_compressed_cache(self, n_per_source, n_sources=None) -> None:
```

Comment:
to download only 1 zip when vali is first starting up, to ensure some data are available